_posts/2019-01-21-workshop.md

```diff
@@ -1,11 +1,16 @@
 ---
 layout: post
-title:  "Spinal Cord MRI Workshop at UCL"
-date:   2019-01-21 13:30:00 GMT+1
+title:  "6th Spinal Cord MRI Workshop"
+date:   2019-01-21 13:30:00 GMT+1 
```

Comment:
i think we can remove the seconds in the time

Reply:
Ok